Changelog_Template.md

```diff
@@ -0,0 +1,46 @@
+This is the template file for changelogs. See the following
+[Confluence page](https://zombie.atlassian.net/wiki/spaces/PRODUCT/pages/2040136662/Create+Changelog+Entries) for the detailed procedure.
+
+TLDR, the following template includes _sample_ entries that you can modify/include
+in the `content/en/Changelog/_index.md` file.
+
+1. If you're including API changes, find the version
+1. Substitute existing changes in each category.
+   1. If there are no changes in a category, delete the title (such as ### Added).
+1. Create a new branch.
+1. Insert the changes in the `content/en/Changelog/_index.md` file, just before the next oldest changelog entry.
```

Comment:
Here, I'd start with creating a branch so that I could copy and edit the template directly on that branch. So this could become something like:

```suggestion
1. Create a branch and insert the new changes in the `content/en/Changelog/_index.md` file, just before the next oldest changelog entry. Use the template below to guide you:

    1. Substitute existing changes in each category.
    1. If there are no changes in a category, remove the category.
```

Reply:
```suggestion
1. Create a new branch.
1. Insert new changes in the content/en/Changelog/_index.md file, just before the next oldest changelog entry.
1. Substitute existing changes in each category.
   1. If there are no changes in a category, remove it by deleting the title.
```